packages/cloud-cognitive/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -318,11 +321,13 @@
 }
 
 .#{$block-class}__head-hidden-select-all {
-  padding-right: $spacing-08;
+  padding-right: $spacing-09;
+
   &.#{$block-class}__select-all-sticky-left {
     position: sticky;
     z-index: 1;
-    left: 0;
+    left: $spacing-09;
+    width: 3.4rem;
```

Comment:
Where is the extra .4rem come from?
It looks like the radios should use a width of 3rem from what I can see.

![image](https://user-images.githubusercontent.com/1717398/220244854-9e1f13c1-6d24-42a0-97e0-c0574b635932.png)

![image](https://user-images.githubusercontent.com/1717398/220244933-65e8d073-6a02-408f-ba7e-c86a32dafb24.png)

Reply:
This is a mistake on my side. I'll fix it in the next commit.